frontend/plugins/frontline_ui/src/modules/integrations/call/components/SipProvider.tsx

```
@@ -774,6 +755,7 @@ const SipProvider = ({
         uaRef.current = null;
       }
     };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)